### ISSUE TITLE
Buienradar API is down, NodeRED keeps restarting because of buienradar

### ISSUE THREAD

**Issue by tverschoof-axians** [dkijkuit/node-red-contrib-buienradar#12]
```
0|node-red  | [xmldom error]    invalid document source
0|node-red  | @#[line:0,col:undefined]
0|node-red  | TypeError: Cannot read property 'nodeType' of undefined
0|node-red  |     at PathExpr.evaluate (/home/thomas/.node-red/node_modules/xpath/xpath.js:1716:17)
0|node-red  |     at XPath.evaluate (/home/thomas/.node-red/node_modules/xpath/xpath.js:1220:25)
0|node-red  |     at XPathExpression.evaluate (/home/thomas/.node-red/node_modules/xpath/xpath.js:4250:26)
0|node-red  |     at Object.exports.selectWithResolver (/home/thomas/.node-red/node_modules/xpath/xpath.js:4667:26)
0|node-red  |     at Object.exports.select (/home/thomas/.node-red/node_modules/xpath/xpath.js:4647:17)
0|node-red  |     at IncomingMessage.<anonymous> (/home/thomas/.node-red/node_modules/node-red-contrib-buienradar/buienradar.js:72:39)
0|node-red  |     at IncomingMessage.emit (events.js:203:15)
0|node-red  |     at endReadableNT (_stream_readable.js:1129:12)
0|node-red  |     at process._tickCallback (internal/process/next_tick.js:63:19)
``` 

It seems like #1 seems the same, so this was fixed at one point, but now it has started happening again.

**Comment by bakman2** [comment]
Same here. Error handling required. If no (correct) data, ignore.

**Comment by dkijkuit** [comment]
Fixed 941a408f82f7e0a1af8d28bdbcf677f09290fea2

New 0.3.3 was relased on NPM.
